openhtf/output/proto/mfg_event_converter.py

```diff
@@ -172,7 +173,7 @@ def _convert_object_to_json(obj):
   # Since there will be parts of this that may have unicode, either as
   # measurement or in the logs, we have to be careful and convert everything
   # to unicode, merge, then encode to UTF-8 to put it into the proto.
-  json_encoder = json.JSONEncoder(sort_keys=True, indent=2, ensure_ascii=False)
+  json_encoder = TestRecordEncoder(sort_keys=True, indent=2, ensure_ascii=False)
```

Comment:
Use the library namespace, though that will require using a different variable for the encoder.